src/Install/InstallServiceProvider.php

```diff
@@ -39,7 +42,7 @@ function () {
                         'json',
                         'mbstring',
                         'openssl',
-                        'pdo_mysql',
+                        implode('|', $drivers->getSupportedPhpDrivers()),
```

Comment:
Well I agree, but I have to admit that I don't see how to do it (I'm pretty new to laravel).

As far as I understand, InstallServiceProvider is used to define services. The problem is that the 'Flarum\Install\Prerequisite\PrerequisiteInterface' needs another service the 'Flarum\Install\Database\Drivers' one.
If I type hint the Driver class in the 'PhpExtensions' constructor, it's not resolved automagically.

What's the best way to do this without refactoring Composite and/or PhpExtensions too heavily?